index.html

```diff
@@ -28,7 +29,7 @@
     <section class="header-page">  
       <div class="header-container">
           <div class="headline">
-            <h1>I’m Emmanuel Glad to see you!</h1>
+            <h1>I’m Emmanuel Glad to see <br/> you!</h1>
             <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 
             <div class="social-media">
```

Comment:
- [ ] As requested by the former code reviewer, which is a requirement for this project, and a best practice to stick to the client's design, please kindly position your social media icons as in the design.  The margin between "LET'S CONNECT" text and the social media icons is too large. Please kindly fix that.

***The design***
![emmyd](https://user-images.githubusercontent.com/65852621/134672270-9ddb14cd-8356-463e-8e82-f53f60068042.png)

***Your implementation***
![emmyI](https://user-images.githubusercontent.com/65852621/134672294-18777a87-cd55-40af-aaa1-ab2510b4380b.png)